package.json

```diff
@@ -14,6 +14,7 @@
     "build": "gulp",
     "lint": "eslint --fix *.js server/**/*.js config/**/*.js && echo Lint Passed ❤",
     "test": "gulp mocha",
+    "lest": "gulp mocha && npm run lint",
```

Comment:
remove the line above this, and run the linter before gulp mocha, it will fix any lint problems before transpiling